oap-server/server-bootstrap/src/main/resources/official_analysis.oal

```diff
@@ -17,22 +17,14 @@
  */
 
 // All scope metrics
-all_p99 = from(All.latency).p99(10);
-all_p95 = from(All.latency).p95(10);
-all_p90 = from(All.latency).p90(10);
-all_p75 = from(All.latency).p75(10);
-all_p50 = from(All.latency).p50(10);
```

Comment:
Regarding pxx, we now use the percentile query at one time. From getMultipleLinearIntValues, we look at one index of the query, but now the back-end index is still divided into five Pxx. This is the next step of transformation? Or is it a merged index that has already been queried?

Reply:
> This is the next step of transformation? Or is it a merged index that has already been queried?

UI will change after this merged. @Fine0830 will do that part.